### PR TITLE
Allow adjustable RSA key length

### LINKED ISSUE #16: Allow adjustable RSA key length
Currently, this bot is issuing a certificate with 4096-bit RSA key by default. Let's Encrypt certbot is issuing 2048-bit RSA key by default (see [here](https://github.com/certbot/certbot/blame/master/README.rst#L150)).

While 4096-bit key offers better security, there are several cases where we need 2048-bit key instead of 4096-bit key. For example, Load balancer in Google Cloud only supports RSA-2048 and ECDSA P-256 certificate (see [here](https://cloud.google.com/compute/docs/load-balancing/http/ssl-certificates)).

It will be nice if there's a mechanism to specify key length from Slack command.

### PR DESCRIPTION
**Pull request by freedomofkeima**
Close #16.

I haven't tested it yet since it requires configuring the initial environment etc 🙇

### CODE REVIEW

File: command/cert.go [hunk 1]
```diff
@@ -34,6 +37,7 @@ type CertService struct {
 	CommonName string
 	Domains    []string
 	CreateKey  bool
+	RSAKeySize string
```
Comment (nabeken) [inline command/cert.go]: Let's accept `int` here then you can convert string to int before you initialize `CertService` struct.

**Comment by nabeken** [comment]
@freedomofkeima Thanks for taking time for this! I incorporated your changes at https://github.com/nabeken/aaa/pull/18 and pushed.

**Comment by freedomofkeima** [comment]
Thanks! 

https://github.com/nabeken/aaa/pull/18/commits/8d690695eef06484795d9f4bbfb6bc2dfdde8ded is cool 😮 From the code structure, does it actually check RSA key size is either 2048 or 4096 from Slack command?

Anyway, thank you so much. This bot is very useful with serverless stuff and now I can incorporate it with Google Load Balancer + GCS!

**Comment by nabeken** [comment]
> From the code structure, does it actually check RSA key size is either 2048 or 4096 from Slack command?

No, it didn't check it. It proved we should check it on `CertService` side. I pushed [the fix](https://github.com/nabeken/aaa/commit/09d9122d6efef8adc3fc479565e8119d7899d458). Thank you~

**Pull request closed without merging**